extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/JPAConfig.java

```diff
@@ -91,8 +70,22 @@ void initDefaultPersistenceUnit() {
         }
     }
 
-    boolean isJtaEnabled() {
-        return jtaEnabled.get();
+    public EntityManagerFactory getEntityManagerFactory(String unitName) {
+        LazyPersistenceUnit lazyPersistenceUnit = null;
+        if (unitName == null) {
```

Comment:
is this condition check useful? I was assuming that `unitName` would be always defined, at most `<default>`.

If I'm wrong, we might want to improve the following (nested) `if` so that the default can still be used/injected even when there's more than one PU defined.

Reply:
It's always set except when you're explicitly calling null. This was the case somewhere in the code base so I decided to make it null safe just to be on the safe side.